src/pages/HomePage/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './ButtonLink.module.scss';
+
+interface ButtonLinkProps {
+  to: string;
+  children: React.ReactNode;
+}
+
+export const ButtonLink: React.FC<ButtonLinkProps> = ({ to, children }) => {
```

Comment:
FC can be imported from React as named import, also is it required to include children in the interface? I mean, isn't this property already implicitly included? Besides that, consider keeping interfaces and types in separate files as it helps to clearly separate concerns